backend/src/test/java/com/daedan/festabook/announcement/service/AnnouncementServiceTest.java

```diff
@@ -26,9 +32,39 @@ class AnnouncementServiceTest {
     @Mock
     private AnnouncementJpaRepository announcementJpaRepository;
 
+    @Mock
+    private OrganizationJpaRepository organizationJpaRepository;
+
     @InjectMocks
     private AnnouncementService announcementService;
 
+    @Nested
+    class createAnnouncement {
+
+        @Test
+        void 성공() {
+            // given
+            AnnouncementRequest request = new AnnouncementRequest(
+                    "폭우가 내립니다.",
+                    "우산을 챙겨주세요.",
+                    true
+            );
+            Long organizationId = 1L;
+            Organization organization = OrganizationFixture.create();
+
+            given(organizationJpaRepository.findById(organizationId))
+                    .willReturn(Optional.of(organization));
```

Comment:
[제안] 사용 순서에 가깝게 순서를 바꾸는 것은 어때요?
제안이라 꼭 반영할 필요는 없어요!

```suggestion
            Organization organization = OrganizationFixture.create();
            Long organizationId = 1L;
            given(organizationJpaRepository.findById(organizationId))
                    .willReturn(Optional.of(organization));
```

Reply:
해당 부분 추후 OrganizationFixture에 필요한 필드라 바로 위에 선언해놨습니다!